teuthology/task/install/rpm.py

```diff
@@ -175,16 +175,16 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
 
     if dist_release in ['opensuse', 'sle']:
         pkg_mng_cmd = 'zypper'
-        pkg_mng_opts = '-n'
-        pkg_mng_gpg_opt = '--no-gpg-checks'
-        pkg_mng_subcommand_opts = '--capability'
-        pkg_mng_install_opts = '--no-recommends'
+        pkg_mng_opts = ['--non-interactive', '--no-gpg-checks']
```

Comment:
I would like to get rid of all those pkg_mng_xrjfoijermfcoriwe variables and use normal values.

Reply:
after adding _flat2gen and couple more variables it becomes very different from the upstream version, and it hard, to merge it.

Reply:
which variables is this adding? I thought it was *removing* variables that are not in the upstream version...

Reply:
_flat2gen is needed - without it, "pkg_mng_opts" (plural) can only handle a single opt